kuka_rox_hardware_interface/src/robot_manager_node.cpp

```diff
@@ -207,6 +241,23 @@ RobotManagerNode::on_deactivate(const rclcpp_lifecycle::State &)
   }
   return SUCCESS;
 }
+
+bool RobotManagerNode::onControlModeChangeRequest(const std::string & control_mode)
+{
+  if(control_mode == POSITION_CONTROL ||
+     control_mode == IMPEDANCE_CONTROL ||
+     control_mode == TORQUE_CONTROL)
```

Comment:
Do not allow torque, will be only supported later

Reply:
Do we actually need this function? This can easily be inlined into the lambda where it's called.